src/hacbs/components/Commits/tabs/CommitsPLRTab.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { Commit } from '../../../types';
+
+type ApplicationOverviewTabProps = {
+  commit: Commit;
+};
+
+const ApplicationOverviewTab: React.FC<ApplicationOverviewTabProps> = ({ commit }) => {
+  return <>PLR of{commit.sha}</>;
+};
+
+export default ApplicationOverviewTab;
```

Comment:
I know this file seems to be temporary, spell out PLR => PipelineRun for consistency.

Also name the component and props to match the file..

Reply:
updated